stock_picking_reorder_lines/models/stock_move.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Alexandre Fayolle
+#    Copyright 2013 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from openerp import fields
+from openerp.models import Model
+
+
+class StockMove(Model):
+    _inherit = 'stock.move'
+    _order = 'date_expected desc, sequence, id'
+
+    sequence = fields.Integer(
+        default=10,
```

Comment:
The default should be calculated dinamically to be the last one plus 1, because this way, when you add more than 10 lines and reorder, and then add another one, you will get this one in between when saving.